www/class/config-generate/abstract/host.class.php

```diff
@@ -37,6 +37,11 @@
 
 abstract class AbstractHost extends AbstractObject
 {
+
+    const VERTICAL_NOTIFICATION = 1;
+    // CLOSE_NOTIFICATION = 2
```

Comment:
not used ?

Reply:
no it's the start point of other method, use by default